geoh5py/ui_json/templates.py

```diff
@@ -404,3 +404,47 @@ def data_value_parameter(
     if optional is not None:
         form.update(optional_parameter(optional))
     return form
+
+
+def drillhole_group_data(
+    value: list[str],
```

Comment:
```suggestion
    value: list[str] | None = [""],
```

Reply:
pylint doesn't like the list as a default value...

Reply:
Yea I guess it should be None by default is optional is possibe.